Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -568,6 +568,10 @@ static String ensureNotNull(const String& text)
     return text;
 }
 
+static GraphicsContextGL::SurfaceBuffer toGCGLSurfaceBuffer(CanvasRenderingContext::SurfaceBuffer buffer)
+{
+    return buffer == CanvasRenderingContext::SurfaceBuffer::DrawingBuffer ? GraphicsContextGL::SurfaceBuffer::DrawingBuffer : GraphicsContextGL::SurfaceBuffer::DisplayBuffer;
```

Comment:
Is the `CanvasRenderingContext::SurfaceBuffer`/`GraphicsContext::SurfaceBuffer` duplication really needed?

Reply:
It's a bit inconvenient, but I'd like to preserve it (GraphicsContextGL is lower level than CanvasRenderingContext, and CanvasRenderingContext cannot depend on GraphicsContextGL). Having a freestanding SurfaceBuffer seems a bit odd and specific and disconnected. If there's a good name, maybe freestanding enum would be the best.